platemate/food/models/common.py

```diff
@@ -50,32 +50,35 @@ def breakdown(self):
         for ingredient in self.measured_ingredients.all():
             boxes[ingredient.box] += [ingredient]
         return boxes
-    
+
     def manual_ingredients(self):
         ingredients = []
         for ingredient in self.measured_ingredients.all():
             if not ingredient.box:
                 ingredients.append(ingredient)
         return ingredients
-    
+
     def mark_processed(self):
         self.processed = datetime.now()
         self.save()
-        
+
     def check_completed(self):
         if self.completed is not None:
             return True
-        
+
+        if self.tagged_boxes is None:
+            return False
```

Comment:
In using check_completed to get the status, I got an error on line 75 with self.tagged_boxes.boxes.